cli: enable `constellation apply` to create new clusters (#2549)
edgelesssys/constellation: docs/docs/getting-started/first-steps-local.md

  This creates a [configuration file](../workflows/config.md) for QEMU called `constellation-conf.yaml`. After that, your current folder also becomes your [workspace](../architecture/orchestration.md#workspaces). All `constellation` commands for your cluster need to be executed from this directory.

2. Now you can create your cluster and its nodes. `constellation apply` uses the options set in `constellation-conf.yaml`.

  ```bash

>   constellation apply

Reviewer: ```suggestion
  constellation apply -y
```

  ```

>   The Output should look like the following::

Reviewer: ```suggestion
  The Output should look like the following:
```

  ```shell-session
  $ constellation apply -y
  Checking for infrastructure changes
  The following Constellation cluster will be created:

>     1 control-plane node of type 2-vCPUs will be created.

Reviewer: ```suggestion
    3 control-plane nodes of type 2-vCPUs will be created.
```
